### PR TITLE
Now works with hapi 8 and the latest moonboots_hapi

### LINKED ISSUE #85: Update templates for hapi version 8

### PR DESCRIPTION
**Pull request by geek**
Closes #85

### CODE REVIEW

File: template/hapi/server.js [hunk 1]
```diff
@@ -18,21 +20,20 @@ server.ext('onPreResponse', function(request, reply) {
...
-        return reply();
-    }
+
+    return reply.continue();
 });
 
 
 // require moonboots_hapi plugin
-server.pack.register({plugin: require('moonboots_hapi'), options: moonbootsConfig}, function (err) {
+server.register({ register: require('moonboots_hapi').register, options: moonbootsConfig }, function (err) {
```
Comment (wraithgar) [inline template/hapi/server.js]: the `.register` after require isn't needed.

**Comment by wraithgar** [comment]
+1 visual w/ only one minor nitpick

File: template/hapi/server.js (hunk 1 above)
Comment (geek) [inline template/hapi/server.js]: It is in hapi 8: http://hapijs.com/api#serverregisterplugins-options-callback

If you change moonboots_hapi to module.exports the register function it won't be though: https://github.com/wraithgar/moonboots_hapi/blob/master/index.js#L43

**Comment by geek** [comment]
Interested in a `.travis.yml`?

File: template/hapi/server.js (hunk 1 above)
Comment (wraithgar) [inline template/hapi/server.js]: "an object or array of objects where each one is either:" ... "an object with the following:" ... "`register` - the plugin registration function."

moonboots_hapi follows the convention in the second bullet point.  If you'll check the server.js example in moonboots_hapi you'll see it sends the whole module, not the register function.

Comment (wraithgar) [inline template/hapi/server.js]: Yep you're right, it works the other way too but the docs show your way.

**Comment by wraithgar** [comment]
My gut feeling is that is more of an [examples](https://github.com/AmpersandJS/examples) addition, this demo app is supposed to just do a quick wireframe so people can see the basics of ampersand code itself.

**Comment by geek** [comment]
Sounds good.  Might as well remove the travis integration with this repo then.

**Comment by fyockm** [comment]
@bear should travis be removed from this repo?
@wraithgar did you mean that this repo should not be updated to hapi 8?

**Comment by bear** [comment]
@fyockm yes, please do remove it - it got added by one of my automation scripts that I've yet tweaked to have per repo exclusions for integrations

**Comment by wraithgar** [comment]
@fyockm =this repo should definitely be updated to hapi8

**Comment by fyockm** [comment]
:+1: this LGTM

@wraithgar we have disabled travis in the AmpersandJS/examples repo, as tests are not required with submitted examples. it might be a good idea to ask for tests with the examples, but I think we didn't want to stifle contribution from the community (at least in the beginning).

**Comment by wraithgar** [comment]
Thank you @geek!

**Pull request merged**